actors/evm/tests/call.rs

```diff
@@ -1031,18 +1013,24 @@ fn make_raw_params(bytes: Vec<u8>) -> Option<IpldBlock> {
     Some(IpldBlock { codec: IPLD_RAW, data: bytes })
 }
 
+const CALL_ACTOR_PRECOMPILE_HEX: &str = include_str!("contracts/CallActorPrecompile.hex");
+
 #[test]
 fn call_actor_solidity() {
-    // solidity
-    let contract_hex = include_str!("contracts/CallActorPrecompile.hex");
     // let mut contract_rt = new_call_actor_contract();
     let contract_address = EthAddress(util::CONTRACT_ADDRESS);
-    let mut tester = ContractTester::new(contract_address, 111, contract_hex);
+    let mut tester = ContractTester::new(contract_address, 111, CALL_ACTOR_PRECOMPILE_HEX);
 
     // call_actor_id
     {
-        let params =
-            CONTRACT.call_actor_id(0, ethers::types::U256::zero(), 0, 0, Bytes::default(), 101);
+        let params = CallActorPrecompile::call_actor_idCall {
```

Comment:
this naming is unfortunate, the compiled contracts have functions suffixed with `Call`, and we can't change that?

Reply:
nevermind, I went and looked at the docs and the answer is _yes_, but I still kind of hate it